www/_data/tools.yml

```
@@ -99,3 +99,12 @@
         feedback solution. With a one minute install guide,
          it enables users to seamlessly report bugs while automatically attaching
         details such as network logs, repro-steps, etc.
+
+-   name: VoltBuilder
+    image: voltbuilder.png
+    url: https://volt.build
+    description: >
+        Modern replacement for PhoneGap. Upload project and certificates,
```

```suggestion
        Modern replacement for PhoneGap Build. Upload project and certificates,
```

Phonegap is not the same as Phonegap Build, VoltBuilder is not a Phonegap replacement